ekglib/sparql/sparql_endpoint.py

```diff
@@ -169,7 +169,7 @@ def execute_sparql_select_query(self, sparql_statement, mime_type=MIME_CSV):
             log_item("Executing", sparql_statement)
         # noinspection PyProtectedMember
         log_item("Statement Type", self.sparql_endpoint._parseQueryType(sparql_statement))
-
+        self.sparql_endpoint.clearCustomHttpHeader("Accept")
```

Comment:
ideally sparql_endpoint should be renamed to "request" and we should not be setting this header when creating the request, so then there would be no need to clear it here...